apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -768,6 +781,12 @@ def grade(element_html: str, data: pl.QuestionData) -> None:
                 else:
                     ans["inner_html"] = None
 
+    # if grading_method is GradingMethodType.ORDERED:
+    #     if partial_credit_type is PartialCreditType.NONE:
+    #         student_answer = [ans["inner_html"] for ans in student_answer]
+    #         true_answer = [ans["inner_html"] for ans in true_answer_list]
+    #         final_score = 1 if student_answer == true_answer else 0
+
```

Comment:
remove commented code

Reply:
Will do.

Reply:
Removed in [4dbe653](https://github.com/PrairieLearn/PrairieLearn/pull/8151/commits/4dbe65319e544be9adbd3c04502415b7bbc9841b).